src/components/Modal.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+type Props = { isActive: boolean; onClick: () => void };
```

Comment:
You will want to use an interface here, the preferred naming convention is IModalProps